pkg/client/query.go

```diff
@@ -0,0 +1,79 @@
+// SPDX-FileCopyrightText: 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package client
+
+import (
+	"github.com/sassoftware/event-provenance-registry/pkg/storage"
+)
+
+// QueryEventByID for querying EPR for Event by ID. Returns nil if no results.
+// fields := []string{"id", "name", "version", "description", "release", "platform_id", "package", "mutipass", "revoked", "timestamp", "eventReceiver_id"}
+func (c *Client) QueryEventByID(id string, fields []string) (*storage.Event, error) {
```

Comment:
Doesn't have to be done right now, but I think we should eventually add support for an "all" field to grab everything.